src/embed/main.js

```diff
@@ -265,6 +293,19 @@ function loop(time) {
     worldRenderer.videoProxy.update(time);
   }
   worldRenderer.render(time);
+  
+  // Update omnitone decoder's rotation matrix.
+  // TODO(hoch): refactor this.
+  if (omnitoneDecoder) {
+    omnitoneRotationMatrix.getInverse(worldRenderer.camera.matrix);
+    var c = omnitoneRotationMatrix.elements;
+    omnitoneDecoder.setRotationMatrix([
```

Comment:
Maybe Omnitone should take a 4x4 rotation matrix natively to skip this conversion step? Also: does roll matter at all?